### PR TITLE
DM-20018

### PR DESCRIPTION
**Pull request by timothydmorton**
This corrects the issue enough to get star/galaxy labels from the `analysisCoaddTable_*` tables (which are single-level).  It doesn't yet update the unit tests to test this more broadly; at least partially because the functor unit tests need to be updated anyway.

### CODE REVIEW

File: python/lsst/qa/explorer/functors.py [hunk 1]
```diff
@@ -121,13 +121,15 @@ def _get_cols(self, parq):
 
         Returns dataframe upon which `self._func` can act.
         """
-        if isinstance(parq, MultilevelParquetTable):
```
Comment (yalsayyad) [inline python/lsst/qa/explorer/functors.py]: This was plenty readable as it was.

File: python/lsst/qa/explorer/functors.py [hunk 2]
```diff
@@ -926,6 +926,7 @@ def dn2mag(self, dn, fluxMag0):
         with np.warnings.catch_warnings():
             np.warnings.filterwarnings('ignore', r'invalid value encountered')
             np.warnings.filterwarnings('ignore', r'divide by zero')
+
```
Comment (yalsayyad) [inline python/lsst/qa/explorer/functors.py:928]: Unnecessary whitespace change

File: python/lsst/qa/explorer/functors.py [hunk 3]
```diff
@@ -882,7 +882,7 @@ def getFilterAliasName(row):
 
 
 class Photometry(Functor):
-    AB_FLUX_SCALE = 3.630780547701013425e12  # AB to NanoJansky (3631 Jansky)
+    AB_FLUX_SCALE = (0 * u.ABmag).to_value(u.nJy)  # AB to NanoJansky (3631 Jansky)
```
Comment (yalsayyad) [inline python/lsst/qa/explorer/functors.py:884]: Seems like an unnecessary dependency, but OK

**Review by yalsayyad — APPROVED** [review]

File: python/lsst/qa/explorer/functors.py (hunk 1 above)
Comment (timothydmorton) [reply python/lsst/qa/explorer/functors.py]: Yeah, thanks. I think I had been planning on using `isMultiLevel` as a variable again somewhere and then changed my mind.
